s5_base/local/prep_samromur_lang.sh

```diff
@@ -5,11 +5,17 @@
 # Description: create the data/local/dict directory to make data/lang for samromur
 # This file should be called by run.sh
 
+if [ $# -ne 3 ] && [ $# -ne 2 ]; then
+  echo "Usage: $0 <text> <out-data-dir> <g2p-model>"
+  echo "e.g.: $0 data/train/text data/ /data/models/g2p/sequitur/althingi/g2p.mdl"
+  echo "      The g2p model is optional."
+  exit 1;
+fi
+
 textfile=$1 # training/text
 data_dir=$2
 dict_dir=$data_dir/local/dict
-# TODO: have g2p as a parameter
-g2p_model=/data/models/g2p/sequitur/althingi/g2p.mdl
+g2p_model=${3:-/data/models/g2p/sequitur/althingi/g2p.mdl}
```

Comment:
I suggest we use Anna's model as the default one, /models/g2p/sequitur/talromur/ipd_clean_slt2018.mdl

Reply:
OK I'll swap it out. 

In a few weeks, i plant to swap out all this sequitur stuff for anna's sampa fairseq models. fyi.